modules/cnv_configuring_masquerade.adoc

```diff
@@ -0,0 +1,35 @@
+[[configuring-masquerade]]
```

Comment:
You'll eventually need to make your module headings reusable and set all the initial headings to H1.

Reply:
Yup, I'm aware that we're lagging behind in that respect. We'll probably do all of them at once.